### DISCUSSION TITLE
Controlling drum speed on a Probatone 12kg

### DISCUSSION THREAD

**Discussion by wgalex85** [artisan-roaster-scope/artisan#1128] [Q&A]
Hello,

I recently updated to the latest version of Artisan and I'm having trouble with controlling the drum speed. 

Any ideea what the problem is ? !Does it have to do with the 'documentation' not beeing filled? 

[image](https://user-images.githubusercontent.com/128070793/225684469-1d12b012-6f71-45b5-bf13-c17bab9715dd.jpeg)

**Comment by bigl00z3** [comment]
Definitely, the Documentation column contains either documentation in some cases or most often when it's an action as Modbus, Phidget or similar, it is the cell where you write the actual command to execute. 
If I load the config in `Config/Machine/Probat/Probatone 7"` and look at the envent page I get:

<img width="904" alt="image" src="https://user-images.githubusercontent.com/8886922/225709030-6f1f0e14-7a75-43a8-9c19-d26d387a5352.png">

so try having in those cells:

- `write(1,12362,{})`
- `write(1,12362,{})`

Nico

**Reply by wgalex85** [reply]
Hey Nico. Thanks for the fast answer. I imputed what you said and it's still not working. Any other ideea why? :) ![image](https://user-images.githubusercontent.com/128070793/225897581-8eec057d-c0bd-45e3-b583-7d5fc7c1cc0e.jpeg)

**Comment by MAKOMO** [comment]
Which version of Artisan are you using? Can you control the burner?

**Reply by wgalex85** [reply]
Latest version, updated yesterday. The burner works fine with all buttons. Also the drum speed can be changed from the roasters display (if it matters)

**Reply by wgalex85** [reply]
2.8.2 is the version to be specific

**Reply by wgalex85** [reply]
![image](https://user-images.githubusercontent.com/128070793/225929391-8733a129-8e61-4dd1-bd54-63878ecf9e92.jpeg)

I imputed as Nico suggested the text in the 'Documentation' colum but still won't work. (Also, when opening up and entering the 'events' tab I noticed the the 5% buttons had the information in the 'Documentation' missing. And it's the seccond time. Is it because I copy pasted the info fron the 10% buttons ? (I didn't know what else to imput and the buttons worked fine afterwards)
